frictionless_ckan_mapper/ckan_to_frictionless.py

```diff
@@ -150,14 +150,49 @@ def dataset(ckandict):
     # package_show can have license_id and license_title
     # TODO: do we always license_id i.e. can we have license_title w/o
     # license_id?
-    if ('licenses' not in outdict and 'license_id' in outdict):
-        outdict['licenses'] = [{
-            'name': outdict['license_id'],
-        }]
-        if 'license_title' in outdict:
-            outdict['licenses'][0]['title'] = outdict['license_title']
+    # There's no mention in the docs of license_title nor license_url, plus
```

Comment:
I am referring to this part of the docs: https://docs.ckan.org/en/2.8/api/index.html#ckan.logic.action.create.package_create

And I'm considering the fields `license_title` and `license_url` as they are part of the package shown by @pdelboca: https://github.com/pdelboca/frictionless-ckan-mapper/commit/ee268c1115f9f19e07c11c9247a4a3cda4eeb236

Because they are not there by default, I treat them all as optional fields (including `license_id`, which is optional according to the docs).